### ISSUE TITLE
reducing the height of the editing area

### ISSUE THREAD

**Issue by janczakb** [GottaYotta/PixelSDK#2]
I would like to increase the image area and reduce the image editing area.

Example: 
![7F36987E-157A-4EBA-B273-AD550B3196CC](https://user-images.githubusercontent.com/4959534/83972929-5ed0b480-a8e3-11ea-888b-1eeacdaa960b.jpeg)

![CA17D608-7C93-472C-9488-E291610D13B6](https://user-images.githubusercontent.com/4959534/83972973-90498000-a8e3-11ea-81ca-86a5c3df5e0e.jpeg)

**Comment by joshbernfeld** [comment]
Thank you for your suggestion. This feature will be included in the next release.

If you are using a ContainerController, you can reduce the height of the controls by including the following line in your EditControllerDelegate call:
```swift
extension ViewController: EditControllerDelegate {
     // Called when the EditController's view will appear.
    func editController(_ editController: EditController, willBeginEditing session: Session) {
        editController.compactControls = true
    }
}
```
or if you are simply initializing an EditController on your own:
```swift
    let session = Session(image: UIImage(named: "test_image")!)
    let editController = EditController(session: session)
    editController.delegate = self
    editController.compactControls = true
```

**Comment by janczakb** [comment]
@joshbernfeld Value of type 'EditController' has no member 'compactControls'.

**Comment by joshbernfeld** [comment]
The member `compactControls` will be available in the next release, which should be coming out soon. I will be sure to comment here when the update is available.

Regards,

Josh

**Comment by joshbernfeld** [comment]
This is now available under the latest release [4.5.0](https://github.com/GottaYotta/PixelSDK/releases/tag/5.4.0)
